extensions/meta-music/package.json

```
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "meta-music",
+  "title": "Meta Music",
+  "description": "Manage the ID3 tags of audio files directly in Raycast!",
+  "icon": "meta-music.png",
+  "author": "teziovsky",
+  "categories": [
+    "Media",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "manage-id3-tags",
+      "title": "Manage ID3 Tags",
+      "description": "Manage the ID3 tags of audio files directly in Raycast!",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "musicDir",
+      "description": "The path to the music folder",
+      "type": "textfield",
```

Maybe you can use `directory` instead
```suggestion
      "type": "directory",
```

---

Yes! Great idea. I didn't know that there was that option.